app/views/foreman_ansible/ansible_roles/_hostgroup_ansible_roles_button.erb

```diff
@@ -1,8 +1,8 @@
 <%=
   play_roles = if hostgroup.all_ansible_roles.empty?
-    "<a title='No Roles assigned' href=\"#\">#{_('Play Roles')}</a>".html_safe
+    "<a title='No Roles assigned' href=\"#\">#{_('Run all Ansible roles')}</a>".html_safe
   else
-    display_link_if_authorized(_('Play Roles'),   hash_for_play_roles_hostgroup_path(:id => hostgroup),   :'data-no-turbolink' => true)
+    display_link_if_authorized(_('Run all Ansible roles'), hash_for_play_roles_hostgroup_path(:id => hostgroup), :'data-no-turbolink' => true)
```

Comment:
We could frase it `on a hostgroup` here as we know we are on a hostgroup :)

Reply:
that seems very long, perhaps add some hover title? That should say something like "Run all Ansible roles on hosts belonging to this host group"

Reply:
Yeah and is not necessary as we are on the hostgroup, so don't worry about this one, title would be nice, but not needed.